### DISCUSSION TITLE
都是谁在卡拉彼丘炸鱼啊

### DISCUSSION THREAD

**Discussion by poikl709** [share121/inter-knot#738] [其他]
天杀的炸鱼√， 我一眼就认出你不是这段位的，快把我的分还给我，你都害我没法好好打游戏了，捅似你喵！捅似你喵！捅似你喵！捅似你喵！捅似你喵！捅似你喵！捅似你喵！捅似你喵！捅似你喵！捅似你喵！捅似你喵！捅似你喵！捅似你喵！捅似你喵！捅似你喵！捅似你喵！捅似你喵！捅似你喵！

**Comment by poikl709** [comment]
还在玩卡拉比丘，我不是说了不要再玩卡拉比丘了喵，我要捅死你喵！捅死你喵！捅死你喵！

**Comment by poikl709** [comment]
天杀的鱼苗，我一眼就认出你是靠ELO上的分！你菜的我都没有游戏体验了！快把我的队友还给我！捅死你喵！捅死你喵！捅死你喵！捅死你喵！捅死你喵！捅死你喵！

**Comment by GanlinSama** [comment]
说完了么？说完了我要冲刺了（doge
